prisma/schema.prisma

```diff
@@ -23,6 +23,7 @@ model Parent {
   first_name         String
   last_name          String
   phone_number       String            @db.VarChar(50)
+  email              String            @unique
```

Comment:
This should be removed from the parent table because we have it in the users table